core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -2315,13 +2324,30 @@ public static PropertyKey fromString(String input) {
   /** Property Key description. */
   private final String mDescription;
 
+  /** Property Key default value. */
+  private final Object mDefaultValue;
+
+  /** Property Key alias. */
+  private final String[] mAliases;
+
   /**
    * @param name String of this property
    * @param description String description of this property key
+   * @param defaultValue default value
+   * @param aliases alias of this property key
    */
-  PropertyKey(String name, String description) {
+  private PropertyKey(String name, String description, Object defaultValue, String[] aliases) {
     mName = Preconditions.checkNotNull(name, "name");
-    mDescription = description != null ? description : "N/A";
+    mDescription = Strings.isNullOrEmpty(description) ? "N/A" : description;
```

Comment:
description should be required IMO. The current code like this is because we do have properties without description filled in, and we should fix it then replace this to null check